Sources/SpeziLicense/Views/ContributionsList.swift

```diff
@@ -0,0 +1,99 @@
+//
+// This source file is part of the Stanford Spezi open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftPackageList
+import SwiftUI
+
+
+/// Present a `ContributionsList` including all used Swift Packages to the user.
+///
+/// The following example shows how to use the ContributionsList:
+/// ```swift
+/// import SpeziLicense
+/// import SwiftUI
+///
+/// struct ExamplePackageDependenciesView: View {
+///
+///     var body: some View {
+///         ContributionsList(appName: "ExamplePackageDependenciesApp", projectLicense: "MIT")
+///     }
+/// }
+/// ```
+public struct ContributionsList: View {
+    var packages: [Package]
+    let appName: String?
+    let projectLicense: String?
```

Comment:
Would be great if we can store the project license as an enum or enum-like struct as discussed in the meeting today.